core/scripts/vrfv2plus/testnet/super_scripts.go

```diff
@@ -231,11 +231,6 @@ func deployWrapperUniverse(e helpers.Environment) {
 	linkAddress := cmd.String("link-address", "", "address of link token")
 	linkETHFeedAddress := cmd.String("link-eth-feed", "", "address of link-eth-feed")
 	coordinatorAddress := cmd.String("coordinator-address", "", "address of the vrf coordinator v2 contract")
-	wrapperGasOverhead := cmd.Uint("wrapper-gas-overhead", 50_000, "amount of gas overhead in wrapper fulfillment")
-	coordinatorGasOverhead := cmd.Uint("coordinator-gas-overhead", 52_000, "amount of gas overhead in coordinator fulfillment")
-	wrapperPremiumPercentage := cmd.Uint("wrapper-premium-percentage", 25, "gas premium charged by wrapper")
-	keyHash := cmd.String("key-hash", "", "the keyhash that wrapper requests should use")
-	maxNumWords := cmd.Uint("max-num-words", 10, "the keyhash that wrapper requests should use")
```

Comment:
Reasoning behind this?

Reply:
This change seems to have been missed out in VRF V2+ Super Script PR and actually seem to break VRF V2+ commands